src/reactapp/src/api/aggregatedQuery/query.js

```diff
@@ -0,0 +1,15 @@
+import { CART_QUERY_PART } from '../cart/fetchGuestCart/query';
+import { COUNTRY_LIST_QUERY_PART } from '../countries/fetchCountryList/query';
+import { CUSTOMER_ADDRESS_LIST_QUERY_PART } from '../customer/fetchCustomerAddresses/query';
+import { CHECKOUT_AGREEMENTS_QUERY_PART } from '../storeConfig/getCheckoutAgreements/query';
+
+export default function getQuery(token) {
```

Comment:
I like to keep it this way. `aggregatedQuery.js` is the starting point of this specific API and it is that function job to prepare everything needed to collect the data. `query.js` purpose is to return the query based on the conditions. These conditions need to be defined by the `aggregatedQuery.js`. Token here falls under that condition. 

Imagine we need to call the aggregated query again from our application after the page loads. Imagine the token is changed before that. Moving the token into the query is not giving us the control. But if we pass it from the `aggregtedQuery.js`, then everything looks good. Because it is `aggregatedQuery.js` responsibility to decide which token needs to pass.

Reply:
Then following this logic why is that in modifier.js u retrieve the token instead of passing it along from aggregatedQuery? This is inconsistent.

Reply:
You are correct. i need to update the modifier in this case